lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-network_manager.rb

```diff
@@ -1,8 +1,10 @@
 module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_NetworkManager < MiqAeServiceExtManagementSystem
-  end
```

Comment:
Thanks @d-m-u.  Before this fix people were saying `create_network_router` had no class.  😄
cc @jrafanie

Reply:
It took you all day to come up with it though huh

Reply:
A little late, but I approve of this joke